content/docs/beyond-the-basics/verifying-with-simple-hash/index.md

```diff
@@ -0,0 +1,132 @@
+---
+ title: "Verifying Assets and Events with Simple Hash"
+ description: "Ensure Asset and Event Data Has Not Changed"
+ lead: "Ensure Asset and Event Data Has Not Changed"
+ date: 2021-05-18T14:52:25+01:00
+ lastmod: 2021-05-18T14:52:25+01:00
+ draft: false
+ images: []
+ menu:
+   docs:
+     parent: "beyond-the-basics"
+ weight: 44
+ toc: true
+---
+
+Verifying your Simple Hash events provides an additional layer of security to your data, so you can ensure the information you have at a given time has not changed.
```

Comment:
Do we want to say "security" in this sentence? Would assurance work better?

Reply:
Agreed, language changed